src/main/java/StringAddCalculator.java

```diff
@@ -0,0 +1,52 @@
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    private static final String DEFAULT_DELIMITERS = ",|:";
+    private static final String CUSTOM_DELIMITER_PATTERN = "//(.)\n(.*)";
+    private static final int CUSTOM_DELIMITER_GROUP_1 = 1;
+    private static final int CUSTOM_DELIMITER_GROUP_2 = 2;
+
+    public static int splitAndSum(String text) {
+        int result = 0;
+        if (text == null || text.isEmpty()) {
+            return result;
+        }
+        String[] strings = splitString(text);
+
+        result = addStrings(strings);
+        return result;
+    }
+
+    private static int addStrings(String[] strings) {
+        int result = 0;
+        for (String string : strings) {
+            int number = numberValidate(string);
+            result += number;
+        }
+        return result;
+    }
+
+    private static int numberValidate(String string) throws RuntimeException {
```

Comment:
```suggestion
    private static int validateNumber(String string) throws RuntimeException {
```

메서드 이름의 경우 동사로 시작하는 것이 관례입니다 :)

Reply:
넵 감사합니다~해당 내용으로 수정하겠습니다